src/js/hooks/cycleCount/useCountStepTable.jsx

```diff
@@ -125,7 +130,7 @@ const useCountStepTable = ({
       const initialValue = _.get(tableData, `[${index}].${columnPath}`);
       const [value, setValue] = useState(initialValue);
 
-      const isFieldEditable = !original.id.includes('newRow') && ![
+      const isFieldDisabled = !original.id.includes('newRow') && ![
```

Comment:
I won't shout at you, but we had a conversation some time ago (about 2 years) about how we should call all of the variables/functions names in a positive manner, so it should be isFieldEditable. But that's of course not a change request.